check-results.js

```diff
@@ -0,0 +1,20 @@
+
+export function didUserWin (userThrow, computerThrow){
+
+    const rps = ["rock", "paper", "scissors"];
```

Comment:
I really like this, and this is what  I would do. Add the options to an array then calling an array method like you did on line 10. Great intuition.